build/rhoda-ci.pod.yaml

```diff
@@ -24,25 +24,25 @@ spec:
 
     - image: <INSERT URL TO YOUR BUILT IMAGE>
       imagePullPolicy: Always
-      name: ods-ci
+      name: rhoda-ci
       env:
         # Use this environment variable to pass args to the ods-ci run script in the container
```

Comment:
The comment should read:
# Use this environment variable to pass args to the rhoda-ci run script in the container